bin/yugabyted

```diff
@@ -1974,6 +2008,16 @@ class YcqlProxy(object):
                 "CREATE KEYSPACE {};".format(keyspace)]
         run_process_checked(cmd)
 
+    # Runs ycqlsh with specified files.
```

Comment:
minor: add an example of the final ycql command line for files a.ycql, b.ycql

Reply:
Okay. will add